pkg/helpers/contexts/heartbeat.go

```diff
@@ -17,12 +17,12 @@ type heartBeatCtx struct {
 	timer    *time.Timer
 	mutex    sync.Mutex
 	lifeTime time.Duration
+	// Cancel cancels the context
+	Cancel context.CancelFunc
```

Comment:
should we make this private?

Reply:
Makes sense to me, added in https://github.com/newrelic/infrastructure-agent/pull/1388/commits/65cbe00029107aad5f151204d6db465d8f7f28f2

Thanks!